JPS_Ontology/ontology/ontobuildingstructure/OntoBuildingStructure.csv

```diff
@@ -0,0 +1,49 @@
+http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,Class,,,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,Quantity
+http://www.ontology-of-units-of-measure.org/resource/om-2/Angle,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,IS-A,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,Angle
+http://www.ontology-of-units-of-measure.org/resource/om-2/Area,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,IS-A,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,Area
+RoofArea,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Area,IS-A,,,,The size of the roof on a building,http://www.theworldavatar.com/kg/ontobuildingstructure/,Roof Area
+SolarSuitableRoofArea,Class,RoofArea,IS-A,,,,The size of the roof area of a building that is suitable for solar use.,http://www.theworldavatar.com/kg/ontobuildingstructure/,Solar Suitable Roof Area
+TotalRoofArea ,Class,RoofArea,IS-A,,,,The total size of the roof of a building.,http://www.theworldavatar.com/kg/ontobuildingstructure/,Total Roof Area 
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasUnit,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,http://www.ontology-of-units-of-measure.org/resource/om-2/Unit,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,has unit
+http://www.ontology-of-units-of-measure.org/resource/om-2/hasValue,Object Property,,,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,has value
+hasQuantity ,"Object Property, Asymmetric Property",,,http://www.w3.org/2002/07/owl#Thing,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,,"Links any entity to a specific quantity defined in the OM ontology. This relationship is not intended to be stated explicitly, but inferred from its sub-properties. For application, only the sub-properties are used.",http://www.theworldavatar.com/kg/ontobuildingstructure/,has quantity
+hasArea,"Object Property, Asymmetric Property",hasQuantity,IS-A,Roof,RoofArea,,A relationship between the roof of a building and the area it covers.,http://www.theworldavatar.com/kg/ontobuildingstructure/,has area of
+hasOrientation,"Object Property, Asymmetric Property",hasQuantity,IS-A,Roof,http://www.ontology-of-units-of-measure.org/resource/om-2/Angle,,A relationship between a roof and its angle/position relative to the Sun.,http://www.theworldavatar.com/kg/ontobuildingstructure/,has orientation
+hasSlope ,"Object Property, Asymmetric Property",hasQuantity,IS-A,Roof,http://www.ontology-of-units-of-measure.org/resource/om-2/Angle,,A relationship between a roof and its slope angle.,http://www.theworldavatar.com/kg/ontobuildingstructure/,has slope
```

Comment:
`hasOrientation` and `hasSlope` should be properties of `InclinedRoof` rather than `Roof` (because they are meaningless for `FlatRoof`).

Reply:
The dataset has these values attached to the Flat Roof as well. However, after investigation, these values for all flat roofs are -1 and 0 respectively. As this data is redundant, I have attach the properties to InclinedRoof instead where there are differences.